README.md

```diff
@@ -141,6 +142,10 @@ A curated list of awesome things related to <a href='https://github.com/vitejs/v
 
 - [vite-plugin-html](https://github.com/anncwb/vite-plugin-html) - Plugin to minimize and use ejs template syntax in `index.html`.
 
+#### SSR
+
+- [vite-plugin-ssr](https://github.com/brillout/vite-plugin-ssr) - Vite SSR Plugin.
```

Comment:
The plugin looks promising! However, I feel the description a bit too general as all the info are already provided in the context. Would you like to update it by telling what's your plugins' features/advantage? Otherwise, I can also merge it as-is if you want.

Reply:
Makes sense, I will update my PR. Sorry for the late reply; GitHub failed to send me an email notification about your comment.

Reply:
Done.